### ISSUE TITLE
NTR: Osmolar gap high in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8265]
New term request Osmolar gap high in blood (elthat_100703164336): Difference from calculated

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000820
- label: Elevated osmolality gap
- parents: [Abnormal serum osmolality (HP:6000679)](https://hpo.jax.org/app/browse/term/HP:6000679)
- pmid: PMID:36000112|PMID:36000112
- def: Plasma osmolality is determined mainly by Sodium, its counter ions, and uncharged species such as Glucose and Urea. Knowledge of the plasma concentration of these species allows calculation of the plasma osmolality quite accurately. If the difference between the measured osmolality (MO) and calculated osmolality (CO) is known as the osmolar gap (OG) is large (greater than 15) osmolar gap can help identify the presence in plasma of substances such as ethanol, methanol, isopropanol, ethylene glycol, propylene glycol..
- comment: High serum osmolarity with an osmolar gap can be yielded by hyperlipidemia or hyperproteinemia, sugars (mannitol, sorbitol), and alcohols.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8265
- synonyms:
